### PR TITLE
Trinity / analyze_diff_expr: fix issue

### PR DESCRIPTION
**Pull request by loraine-gueguen**
fix issue https://github.com/galaxyproject/tools-iuc/issues/1275

### CODE REVIEW

**Comment by abretaud** [comment]
Cool, thanks!
Can you bump the tool version?

**Comment by loraine-gueguen** [comment]
Yep. Done

File: tools/trinity/analyze_diff_expr.xml [hunk 1]
```diff
@@ -83,6 +83,7 @@
         </collection>
         <data format="RData" name="rdata" label="${tool.name} on ${on_string}: RData file" from_work_dir="results.matrix.RData"/>
         <collection name="GOseq_enrichment" type="list" label="${tool.name} on ${on_string}: GOseq enriched and depleted categories">
+            <filter>GO_enrichment['examine_GO_enrichment'] == 'yes'</filter>
```
Comment (abretaud) [inline tools/trinity/analyze_diff_expr.xml]: Shouldn't it be `additional_params['GO_enrichment']['examine_GO_enrichment']`?

Comment (loraine-gueguen) [inline tools/trinity/analyze_diff_expr.xml]: You're right. I was not sure if the section name should be included.

**Comment by abretaud** [comment]
Green! Merci !

**Comment by nsoranzo** [comment]
This was automatically updated only on the TTS because only one tool of the suite has changes, so the exit code of `planemo shed_update` was not 0: https://travis-ci.org/galaxyproject/tools-iuc/jobs/226432110
@jmchilton suggestions? I'm not sure if this is a bug.

**Pull request merged**